app/test/frontend/golden/pkg_show_version_page.html

```diff
@@ -233,8 +233,6 @@ <h3 class="title">Uploader</h3>
                 <a href="/packages?q=email%3Ahans%40juergen.com" title="Search packages from hans@juergen.com" rel="nofollow">hans@juergen.com</a>
               </span>
             </p>
-            <h3 class="title">License</h3>
-            <p>BSD (LICENSE.txt)</p>
```

Comment:
Why is the license just gone here?

Reply:
Change in how the data is extracted. In the non-migrated tests (those without the test-profile) this is broken because inconsistencies in the data, but in the other case (those with migrated to test-profile) this is consistent, and thus it appears as new link.